buku.py

```diff
@@ -451,6 +451,17 @@ def initdb(dbfile=None):
 
         return (conn, cur)
 
+    def get_rec_all(self):
```

Comment:
Please rename to `get_all_rec()`.
Just to maintain the same pattern used in other APIs.